src/plugins/intel_gpu/tests/unit/test_cases/fully_connected_gpu_test.cpp

```diff
@@ -1333,7 +1333,7 @@ class fully_connected_gpu_tests: public ::testing::Test {
         auto config = get_test_default_config(engine);
         config.set_property(ov::intel_gpu::allow_new_shape_infer(true));
         config.set_property(ov::intel_gpu::optimize_data(true));
-        config.set_property(ov::hint::dynamic_quantization_group_size(32));
+        config.set_property(ov::hint::dynamic_quantization::group_size(32));
```

Comment:
should we deprecate old one?

Reply:
Could you elaborate your intention of deprecation? Are you suggesting to remove the old API itself?

Reply:
I supposed that what you are proposing in current PR is replacement for old `dynamic_quantization_group_size`, is not it?

Reply:
Yes, it is deprecating old one. But just as other deprecation, I think we need to keep old one too because it is already publicly available API. That is why it is adding new hint while keeping the old one as-is.

Reply:
but deprecate != remove.
I think you can deprecate in current PR and remove in several releases.